test/js/loan-comparison-api_test.js

```diff
@@ -0,0 +1,86 @@
+var chai = require('chai');
+var expect = chai.expect;
+var sinon = require('sinon');
+
+var api;
+
+describe('Loan comparison API tests', function() {
+  before(function() {
+    api = require('../../src/static/js/modules/loan-comparison/api.js');
+  });
+
+  describe('Fetch county data', function() {
+    it('should fetch county data', function() {
+      var arg = {state: 'DC'};
+      api.fetchCountyData(arg);
+      // TODO: don't know yet how to spy/stub jQuery
```

Comment:
Do you plan to do this at a later PR?  Or in this PR?

You can look at promise and deferred in loan-store tests, or something like here: http://stackoverflow.com/questions/13148356/how-to-properly-unit-test-jquerys-ajax-promises-using-jasmine-and-or-sinon